archive_extractor/zip_archiver.go

```diff
@@ -11,6 +11,9 @@ import (
 )
 
 const fileHeaderSignatureString = "PK\x03\x04"
+const zoneInfoFileHeaderSignatureString = "\x23\x20\x76\x65\x72\x73\x69\x6F\x6E"
+
+var ErrZoneInfoFile = errors.New("zone info file found instead of zip")
```

Comment:
I think it's better define this new error as a struct that implement the error interface.
Once we have an dedicated type for this error, we can use type assertion on the calling function to easily identify this kind of error.